fdbserver/DataDistribution.actor.cpp

```diff
@@ -2462,6 +2462,14 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 			traceAllInfo();
 		}
 
+		for (int t = 0; t < badTeams.size(); t++) {
```

Comment:
In case this creates slow task, we can add `badTeams` field into TCTeamInfo and assign badTeam to its server's TCTeamInfo. Then we can quickly cancel it.